src/createRollupConfig.ts

```diff
@@ -186,18 +195,20 @@ export async function createRollupConfig(
       //   printInfo: false,
       // }),
       shouldMinify &&
-        terser({
-          sourcemap: true,
-          output: { comments: false },
-          compress: {
-            keep_infinity: true,
-            pure_getters: true,
-            passes: 10,
-          },
-          ecma: 5,
-          toplevel: opts.format === 'cjs',
-          warnings: true,
-        }),
+        (opts.closureCompiler
+          ? compiler(closureCompilerOptions)
```

Comment:
Sorry, maybe this wasn't clear in my last review -- let's just add default options in here, you had `ADVANCED_OPTIMIZATIONS` before. If a user wants to override this they can do that by overriding `config.plugins` in `tsdx.config.js` as with other plugins.

That's not to say that it's a bad idea to allow for easier configuration of plugins in `tsdx.config.js`, it's just that we haven't done it yet and I don't want to introduce a new API without thinking through it more, especially not for an experimental option.

I think a future API for easier configuration of plugins should probably reflect something like `customize-cra` -- I think I mentioned this in #389

Reply:
The main goal is to add ability (not for all, but for core maintainers :)  to start playing with closure compiler as internal experimental feature and collect right options to become defaults.. (this is why I've used dedicated config file in initial commit).

Reply:
Yea I know you mentioned for internal usage, I didn't know you meant for figuring out defaults.

I think the problem with libraries is that once you expose an API, you shouldn't break it too easily and should remain consistent. Hence my resistance to exposing one at all.

Maintainers / internal users would know how to adjust `createRollupConfig.ts` and `tsdx.config.js` so I don't think that's an issue.
It is marked as experimental, so I'm not worried about changing the defaults too much in that sense. We could _also_ say it's "UNSTABLE" to really get the point across 😅

Reply:
using
```js
closureCompiler({
   compilation_level: 'SIMPLE_OPTIMIZATIONS',
 })
```
 as defaults